src/tallies/tally_scoring.cpp

```diff
@@ -317,7 +317,7 @@ score_fission_eout(const Particle* p, int i_tally, int i_score, int score_bin)
 //! is not used for analog tallies.
 
 void
-score_general_ce(const Particle* p, int i_tally, int start_index,
```

Comment:
Why is it that this and other tally functions no longer use a `const` pointer to the particle?

Reply:
There's a call to `data::nuclides[i_nuclide]->calculate_elastic_xs(Particle&)` which now modifies the particle since it is storing the microscopic xs cache.